sopel/irc/utils.py

```diff
@@ -51,3 +55,13 @@ def nop(bot, cap):
         self.arg = arg
         self.failure = failure or nop
         self.success = success or nop
+
+
+class MyInfo(collections.namedtuple('MyInfo', MYINFO_ARGS)):
+    """Store client, servername, and version from ``RPL_MYINFO`` events.
+
+    .. seealso::
+
+        https://modern.ircdocs.horse/#rplmyinfo-004
```

Comment:
```suggestion
        https://modern.ircdocs.horse/#rplmyinfo-004

```

Pretty much everywhere else, the indented text below `.. seealso::` is surrounded by blank lines both above and below.